packages/neuron-wallet/src/controllers/wallets.ts

```diff
@@ -166,6 +167,19 @@ export default class WalletsController {
       throw new InvalidJSON()
     }
     const keystoreObject = Keystore.fromJson(keystore)
+
+    if (keystoreObject.isFromCkbCli()) {
+      throw new UnsupportedCkbCliKeystore()
+    }
+
+    try {
+      keystoreObject.extendedPrivateKey(password)
+    } catch (error) {
+      if (error.message === 'Incorrect password!') {
+        throw new IncorrectPassword()
+      }
```

Comment:
The `else` branch should be handled, otherwise, the error will be ignored if the message is not `Incorrect password!`. BTW, it's better to match an error code instead of the message.

Reply:
The error from @ckb-lumos only contains the message, without a specific error code.

https://github.com/ckb-js/lumos/blob/f2f5666f0f8e9e0bf5aec4023abecd0a6c0ddc8b/packages/hd/src/keystore.ts#L18

Reply:
![image](https://github.com/nervosnetwork/neuron/assets/11756869/a8214433-6d82-48a0-aec7-7b625f40434c)

Reply:
Got it